akka-http-core/src/main/java/akka/http/javadsl/model/FormData.java

```diff
@@ -51,4 +53,14 @@ public static FormData create(Pair<String, String>... params) {
   public static FormData create(Map<String, String> params) {
     return new FormData(Query.create(params));
   }
+
+  /**
+   * Creates a FormData from the given parameters.
+   */
+  public static FormData create(Iterable<Pair<String, String>> params) {
+    @SuppressWarnings("unchecked")
+    final Pair<String, String>[] paramsArray =
+          StreamSupport.stream(params.spliterator(), false).toArray(Pair[]::new);
```

Comment:
Totally correct! Will do this! I just copied from Query.create(Iterable...), but I can call this instead.